plugins/org.eclipse.fordiac.ide.export.forte_ng/src/org/eclipse/fordiac/ide/export/forte_ng/base/BaseFBImplTemplate.xtend

```diff
@@ -57,24 +59,18 @@ abstract class BaseFBImplTemplate<T extends BaseFBType> extends ForteFBTemplate<
 		«ENDIF»
 		«IF !type.internalConstVars.isEmpty»
 			«type.internalConstVars.generateVariableDefinitions(true)»
-			
-		«ENDIF»
-		«IF !type.internalFbs.isEmpty»
-			«type.generateInternalFbDeclarations»
-			
 		«ENDIF»
+		
 		«FBClassName»::«FBClassName»(const CStringDictionary::TStringId paInstanceNameId, forte::core::CFBContainer &paContainer) :
 		    «baseClass»(paContainer, &scmFBInterfaceSpec, paInstanceNameId, «IF !type.internalVars.empty»&scmInternalVars«ELSE»nullptr«ENDIF»)«// no newline
+		    			»«generateInternalFBInitializer»«// no newline
 		    			»«(type.internalVars + type.interfaceList.inputVars + type.interfaceList.inOutVars + type.interfaceList.outputVars).generateVariableInitializer»«// no newline
 		    			»«(type.interfaceList.sockets + type.interfaceList.plugs).generateAdapterInitializer»«generateConnectionInitializer» {
```

Comment:
I'm confused, why is this space not exported?

Reply:
The space (and the newline) is inside a `«...»` block.